### PR TITLE
Cookbooks files movearound

### PR DESCRIPTION
**Pull request by alarshi**
This is a PR to address issue #3657 for the Burnman cookbook.

### Before your first pull request:

* [x ] I have read the guidelines in our [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) document.

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### CODE REVIEW

**Comment by tjhei** [comment]
The documentation doesn't compile but fails with 
```make: *** No rule to make target 'manual/cookbooks/burnman/doc/adiabatic_conditions.part.prm.out', needed by 'manual.pdf'.  Stop.
```
I think your symbolic link should be to the parent directory (and not to /doc/).

File: cookbooks/burnman/doc/burnman.tex [hunk 1]
```diff
@@ -15,25 +16,28 @@ \subsubsection{2D compressible convection with a reference profile and material
 to generate the reference profile has the advantage that all the material property data
 are consistent, for example, the gravity profile is computed using the reference density.
 \begin{figure}
-  \includegraphics[width=\textwidth]{./reference_profile.pdf}
+  \includegraphics[width=\textwidth]{../../cookbooks/burnman/doc/reference_profile.pdf}
```
Comment (tjhei) [inline cookbooks/burnman/doc/burnman.tex:19]: I think you can also write this and all other places as 
```suggestion
  \includegraphics[width=\textwidth]{cookbooks/burnman/doc/reference_profile.pdf}
```

Can you try?

**Comment by tjhei** [comment]
This looks good (and compiles). I will let you check if the simpler path also works. Otherwise, this is good to go.

File: cookbooks/burnman/doc/burnman.tex (hunk 1 above)
Comment (alarshi) [reply cookbooks/burnman/doc/burnman.tex:19]: Yes, I will try that and also remove some past commits. I was just wondering that the path is relative to the manual folder, so if we remove the linked list, this path would not work, or am I understanding it incorrectly?

**Comment by gassmoeller** [comment]
Tests have finished. Thanks!

**Pull request merged**